app/graphql/loaders/fancy_loader/rank_query_generator.rb

```diff
@@ -0,0 +1,32 @@
+# @private
+class Loaders::FancyLoader::RankQueryGenerator
+  # @param key [Symbol] The table row key
+  # @param partition_key [Symbol] The find_by key for the table
+  # @param table [Arel::Table]
+  def initialize(key, partition_key, table)
+    @key = key
```

Comment:
I don't think `key` is the right name for this, how about

```suggestion
  # @param column [Symbol] The column to rank by
  # @param partition_key [Symbol] The find_by key for the table
  # @param table [Arel::Table]
  def initialize(column, partition_key, table)
    @column = column
```